_profile/prateek.md

```diff
@@ -0,0 +1,26 @@
+---
+# This template adds a profile to hacktoberfest. Checkout sample profile at
+# https://github.com/DotSlashCommunity/hacktoberfest/blob/master/_profile/ksdme.md
+
+# Required
+layout: page
+title: profile
+
+# Required
+name: Prateek Tiwari
+
+# Optional, atleast 2 recommended
+email: nopanictri@yahoo.com
+quora: <Your Quora profile>
+google: <Your Google+ profile>
+github: <Your GitHub profile> 
+youtube: <Your YouTube profile>
+twitter: <Your Twitter profile>
+linkedin: <Your LinkedIn profile>
+instagram: <Your Instagram profile>
+
+# Optional
+# Upload the image to imgur and
+# paste the direct link to image
+picture: https://imgur.com/a/EdaOCpp
```

Comment:
You linked to imgur page, you should instead link to the image  i.e https://i.imgur.com/EmNts9B.jpg